pom.xml

```diff
@@ -25,6 +25,10 @@
 
     <properties>
         <mockito.version>1.9.5</mockito.version>
+		<progman-client.version>0.0.4-SNAPSHOT</progman-client.version>
```

Comment:
So... it *looks* consistent in my IDE, but there's a good mix of tab and space indenting in this POM file.  Do we have a standard?